packages/askui-nodejs/src/core/ai-element/ai-elements-options.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Options for configuring how AI elements are collected.
+ *
+ * @interface AIElementOptions
+ *
+ * @property {string[]} sourceDirectories - List of source directories to search for AI elements.
+ *  These directories will be checked for the presence of AI element files.
+ *
+ * @property {'Warn' | 'Error' | 'Ignore'} notFoundAction - Action to take when
+ *  one or more source directories are not found.
+ *      - `'Warn'` → Logs a warning if a directory is missing.
+ *      - `'Error'` → Throws an error if a directory is missing.
+ *      - `'Ignore'` → Skips missing directories without logging anything.
+ *  This action is triggered when specified directories do not exist,
+ *      not when AI elements within those directories are missing.
+ */
+export interface AIElementOptions {
+  sourceDirectories: string[]
```

Comment:
Can we rename this to additionalLocations

```suggestion
  additionalLocations: string[]
```